--- src/AAPlanetPerihelionAphelion2.h	2020-06-01 22:21:58.000000000 +0200
+++ src/AAPlanetPerihelionAphelion2.h.cc	2020-11-09 12:52:37.051318700 +0100
@@ -44,7 +44,11 @@
 {
 public:
 //Enums
+#ifdef CC_CXX11_ENUM_CLASS
   enum class Type
+#else
+  enum Type
+#endif
   {
     NotDefined = 0,
     Perihelion = 1,
@@ -75,7 +79,11 @@
 {
 public:
 //Enums
+#ifdef CC_CXX11_ENUM_CLASS
   enum class Object
+#else
+  enum Object
+#endif
   {
     MERCURY,
     VENUS,
