--- src/AARiseTransitSet2.h	2019-07-15 11:36:54.000000000 +0200
+++ src/AARiseTransitSet2.h.cc	2019-09-04 15:06:27.602026200 +0200
@@ -44,7 +44,11 @@
 {
 public:
 //Enums
+#ifdef CC_CXX11_ENUM_CLASS
   enum class Type
+#else
+  enum Type
+#endif
   {
     NotDefined,
     Rise,
@@ -81,7 +85,11 @@
 {
 public:
 //Enums
+#ifdef CC_CXX11_ENUM_CLASS
   enum class Object
+#else
+  enum Object
+#endif
   {
     SUN,
     MOON,
